srcpkgs/wabt/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'wabt'
+pkgname=wabt
+version=1.0.19
+revision=1
+build_style=cmake
+short_desc="WebAssembly Binary Toolkit "
+maintainer="Isaac Freund <ifreund@ifreund.xyz>"
+license="Apache-2.0"
+homepage="https://github.com/WebAssembly/wabt"
+_wasm_c_api_commit=d9a80099d496b5cdba6f3fe8fc77586e0e505ddc
+distfiles="https://github.com/WebAssembly/wabt/archive/${version}.tar.gz
+ https://github.com/WebAssembly/wasm-c-api/archive/${_wasm_c_api_commit}.tar.gz"
+checksum="134f2afc8205d0a3ab89c5f0d424ff3823e9d2769c39d2235aa37eba7abc15ba
+ aea8cd095e9937f1e14f2c93e026317b197eb2345e7a817fe3932062eb7b792c"
+
+pre_configure() {
+	# copy submodule to proper location
+	cp -rup ${XBPS_BUILDDIR}/wasm-c-api-*/* ${wrksrc}/third_party/wasm-c-api
```

Comment:
Please rather move like in `tracebox`, otherwise ${XBPS_BUILDDIR}/wasm-c-api won't be cleaned.

Reply:
Will fix for this and `wasmtime`, I just copied blindly from `ppsspp` which I will also fix in a minute then.